plutus-core/untyped-plutus-core-test/Evaluation/Machines/Budget/IfThenElse/5.plc.golden

```diff
@@ -1,13 +1,32 @@
-( (Left An error has occurred:  User error:
-The limit ({ cpu: 1000
-| mem: 1000
-}) was reached by the execution environment. Final state: ({ cpu: 0
-| mem: 1297
-}))
-, ({ tally: ({ BAST causes ({ cpu: 0
+( (Right (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> f (delay (\z -> \f -> z)))))))))))))))))))))))
+, ({ tally: ({ BConst causes ({ cpu: 5
+| mem: 0
```

Comment:
Previously we had two budgeting modes:

1. tallying + counting
2. tallying + restricting

Now the second one is gone (which is why this golden test is different), 'cause I don't find it very useful (feel free to insist on keeping it), but two more are added, giving us

1. tallying + counting
2. counting
3. restricting

`1` is to keep tests the same, not sure if we actually need to do tallying and counting at the same time.

Reply:
No regrets to seeing tallying+restricting gone.  Tallying+counting is convenient in cost model experiments because it lets you  see everything that's going on.